src/sdltiles.cpp

```diff
@@ -2240,6 +2240,16 @@ void play_music(std::string playlist) {
 #endif
 }
 
+void update_music_volume( const int volume ) {
+#ifdef SDL_SOUND
+    Mix_VolumeMusic( volume * get_option<int>( "MUSIC_VOLUME" ) / 100 );
+#endif
```

Comment:
You need to add a `(void)volume` in an `#else` block, otherwise there will be an unused argument warning when compiling without sound.